src/client/src/Stream.c

```diff
@@ -712,8 +712,14 @@ STATUS logStreamMetric(PKinesisVideoStream pKinesisVideoStream)
     // V2 stream information
     DLOGD("\tCurrent elementary frame rate (fps): %f ", streamMetrics.elementaryFrameRate);
 
+    // V3 stream information
+    DLOGD("\tAPI Call Retry Count : %llu", streamMetrics.streamApiCallRetryCount);
+
     // V1 client information
     DLOGD("\tTotal elementary frame rate (fps): %f ", clientMetrics.totalElementaryFrameRate);
```

Comment:
`%lf` for clientMetrics.totalElementaryFrameRate as well since its also `DOUBLE`